app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -1468,7 +1471,7 @@ public boolean canOpenNewWindow() {
     }
 
     @Override
-    public void openNewWindow(String uri) {
+    public void openNewWindow(@NonNull String uri) {
```

Comment:
This is being called from https://github.com/MozillaReality/FirefoxReality/blob/2516a84dcc5b425a9178dc3f441938ce57e8d2b6/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/menus/ContextMenuWidget.java#L84 with a `Nullable `string.

Reply:
All of the `@NonNull` additions were because the base class has it set. So we either need to fix the base class or fix our usage of it.